### PR TITLE
fitness check done

### PR DESCRIPTION
**Pull request by Kellsonphilips**

### CODE REVIEW

File: reports/kellsonphilips/Second-Task-start/Fitness-Management-Record/morning-fitness-routine.md [hunk 1]
```diff
@@ -92,3 +92,16 @@
 - Neck exercise 12 in 4x reps : 48      
 - Tricep exercise 20 in 5x reps : 100 
 <hr>
+
+## Date: 28/10/2022
+ - [x] Check if satified
```
Comment (avie-dev) [inline reports/kellsonphilips/Second-Task-start/Fitness-Management-Record/morning-fitness-routine.md]: I think I have to fix this now.. change satisfied to "satisfied" haha

**Review by avie-dev — CHANGES_REQUESTED** [review]

File: reports/kellsonphilips/Second-Task-start/Fitness-Management-Record/morning-fitness-routine.md (hunk 1 above)
Comment (Kellsonphilips) [reply reports/kellsonphilips/Second-Task-start/Fitness-Management-Record/morning-fitness-routine.md]: haha thanks for pointing it out. I will change it in another PR please 😂

**Comment by Kellsonphilips** [comment]
You can help me edit @avie-dev You have my permission

**Review by Kellsonphilips — COMMENTED** [review]
All typos have been corrected @avie-dev

**Pull request merged**